android/res/values/strings.xml

```diff
@@ -21,7 +21,16 @@
 <resources>
     <!-- Important: Android is buggy with HTML, do not put HTML tags right next to text, leave a space, as it seems the text wrapping logic is considering HTML as visible text and you get premature wrapping. -->
     <string name="your_privacy_is_a_fundamental_right_html">Your privacy is a fundamental human right.</string>
-    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[<p> * Although highly recommended, <b>you do not need a VPN to use FrostWire. </b></p>]]></string>
+    <string name="you_dont_need_a_vpn_to_use_frostwire_bullet_html"><![CDATA[* Although recommended for your downloads, <b>you do not need a VPN to use FrostWire. </b>]]></string>
```

Comment:
Ideally, you should not have formatting inside the string resources, strings res are more for data (minor formatting like \n is aceptable), but if you look at `VPN_feature_list_html`, that's is a higher level of formatting.

The use of CDATA (together with the comments by angel) is a strong indication that this is not the best way. If you use raw resources, that's a clear way to say that you need to put a lot of stuff that does not fit in a short single line <string> concept.

Can you explain to me the idea of `protected_connections_visibility_bullet_html`. If you are using a VPN, are you still recommending another one?